### PR TITLE
Use new PrePostProcessing for legacy MO frontends

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Replace legacy MO preprocessing with new ngraph based one*

### Tickets:
 - *73719*

### CODE REVIEW

File: tools/mo/openvino/tools/mo/back/ReverseInputChannels.py [hunk 1]
```diff
@@ -536,7 +475,7 @@ def find_and_replace_pattern(self, graph: Graph):
 
 class ApplyReverseChannels(BackReplacementPattern):
```
Comment (jane-intel) [inline tools/mo/openvino/tools/mo/back/ReverseInputChannels.py:476]: As I understand (it is not mentioned anywhere!) this code is left here to fuse internal RIC operator for TensorFlow models with Pack-Unpack pattern. So, why can we adopt current logic by @GlebKazantaev and delete this file completely?

**Review by jane-intel — COMMENTED** [review]
How was this tested from performance point of view?

File: tools/mo/openvino/tools/mo/back/ReverseInputChannels.py (hunk 1 above)
Comment (mvafin) [reply tools/mo/openvino/tools/mo/back/ReverseInputChannels.py:476]: You are right, I don't think we should delete it.

**Review by vgavrilo — APPROVED** [review]

**Review by rkazants — CHANGES_REQUESTED** [review]
Please do not merge. I observe multiple failures from DLB results. I will share about details by email

**Review by evolosen — APPROVED** [review]

**Pull request closed without merging**